packages/components/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base.json",
```

Comment:
Each project's tsconfig.json extends tsconfig.base.json (so all packages share similar configuration), sets outDir, rootDir, and include options (since those are resolved relative to the config file, and adds any package specific configuration (in this case, noImplicitAny: false).